test/Dialect/Torch/canonicalize.mlir

```diff
@@ -1062,6 +1062,16 @@ func.func @torch.aten.remainder.int() -> !torch.int {
     return %ret : !torch.int
 }
 
+// CHECK-LABEL:   func.func @torch.aten.pow.int_float() -> !torch.float {
+// CHECK:           %[[FLOAT_8:.*]] = torch.constant.float 8.000000e+00
+// CHECK:           return %[[FLOAT_8]] : !torch.float
+func.func @torch.aten.pow.int_float() -> !torch.float {
```

Comment:
This should also be tested e2e (I'm guessing you were running into the same overload issue as with the other PRs